command/job_inspect.go

```diff
@@ -34,10 +34,17 @@ General Options:
 Inspect Options:
 
   -version <job version>
-    Display the job at the given job version.
+    Display the job at the given job version. Defaults to current version.
 
   -json
-    Output the job in its JSON format.
+    Output the job in its JSON format. Cannot be used with -hcl.
+
+  -hcl
+    Output the original HCL submitted with the job. Cannot be used with -json.
+
+  -with-vars
+    Include the original HCL2 variables submitted with the. Can only be used
```

Comment:
with the what?

Reply:
I wanted to leave you in suspense. :grin: Will fix.

Reply:
That makes me realize I forgot docs and a changelog. Will fix that too.